src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -360,6 +376,30 @@ const formatAuthenticatorOptions = (
         AUTHENTICATOR_KEY.PHONE,
       ];
 
+      // @ts-ignore TODO: Add grace period fields to auth-js SDK https://oktainc.atlassian.net/browse/OKTA-848910
+      const date = new Date(authenticator?.gracePeriod?.expiry);
+      const gracePeriodEpochTimestampMs = !Number.isNaN(date.getTime()) ? date.getTime() : 0;
```

Comment:
You may be able to make this more succinct by changing the const to
```js
const dateMs = new Date(authenticator?.gracePeriod?.expiry).getTime();
```
And then you only need to call `getTime()` once

Reply:
Separately, prefer positive logic for readability. You can invert the ternary to avoid the `!` negation.